src/app/services/baw-api/baw-api.service.ts

```diff
@@ -167,16 +187,14 @@ export class BawApiService<
   public list(
     classBuilder: ClassBuilder,
     path: string,
-    opts?: CacheOpt & NotificationsOpt
+    options: BawServiceOptions = defaultBawServiceOptions
```

Comment:
there's not much reason to set this to the default if you're just merging it in with the default one line later.

Would ` options: BawServiceOptions = {}` do instead?

Reply:
Good point, I'll make the code change